prophesee_pin_config:
  evc3a_plugin_gen31:
    external: 0
    loopback: 6
  hal_plugin_gen31_evk2:
    external: 0
    loopback: 3
  hal_plugin_gen31_evk3:
    external: 0
    loopback: 6
  hal_plugin_gen31_fx3:
    external: 0
    loopback: 6
  hal_plugin_gen3_fx3:
    external: 0
    loopback: 6
  hal_plugin_gen41_evk2:
    external: 1
    loopback: 3
  hal_plugin_gen4_evk2:
    external: 1
    loopback: 3
  hal_plugin_gen4_fx3:
    external: 1
    loopback: 6
